CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [1.0.1] - 2022-04-11
+
+### Fixed
+
+- Fixed a regression where export, by default, export dependencies for all non-optional groups instead of just the default group. [#50](https://github.com/python-poetry/poetry-plugin-export/pull/50)
+
```

Comment:
I did not check this in real, but from what I observed in the unit tests I think in `1.0.0` no other groups than the default group were exported (no matter which options used) because `get_project_dependency_packages()` has been called with `dev=False`. (I suppose this bug covered up the other bug of changed default behavior.)

Reply:
Fixed.